src/test/java/com/datapipe/jenkins/vault/it/VaultTokenCredentialBindingIT.java

```diff
@@ -38,11 +38,12 @@ public void shouldInjectCredentialsForAppRole() {
         story.addStep(new Statement() {
             @Override
             public void evaluate() throws Throwable {
-                VaultAppRoleCredential c = mock(VaultAppRoleCredential.class);
-                when(c.getToken(any(Auth.class))).thenReturn(token);
-                when(c.getId()).thenReturn(credentialsId);
+                VaultAppRoleCredential c = new VaultAppRoleCredential(CredentialsScope.GLOBAL,
```

Comment:
@jetersen I had to fix this IT test which I think broke due to credential binding plugin update. It might need to be back ported to #220 

I also noticed that the Jenkins build check is still trying to run against 2.277.1 even though the Jenkinsfile is updated, I assume that's because it doesn't trust the PR Jenkinsfile. Not sure what the process is to address that failure.

Reply:
Thanks for the heads up I have adapted the IT tests

Yes, only those with write access can modify the Jenkinsfile otherwise Jenkins simply won't trust you due to security/abuse concern 😅